content/en/docs/concepts/certificaterequest.md

```diff
@@ -72,170 +97,498 @@ Each ready condition consists of the pair `Ready` - a boolean value, and
 
 ```yaml
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: cert-manager-controller-approve:my-issuer-example-com # edit
+  name: cert-manager-controller-approve:google-cas-issuer # edit
 rules:
-- apiGroups:
-  - cert-manager.io
-  resources:
-  - signers
-  verbs:
-  - approve
-  resourceNames:
-  - issuers.my-issuer.example.com/* # edit
-  - clusterissuers.my-issuer.example.com/* # edit
+  - apiGroups:
+      - cert-manager.io
+    resources:
+      - signers
+    verbs:
+      - approve
+    resourceNames:
+      - googlecasissuers.cas-issuer.jetstack.io/*
+      - googlecasclusterissuers.cas-issuer.jetstack.io/*
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: cert-manager-controller-approve:my-issuer-example-com # edit
+  name: cert-manager-controller-approve:google-cas-issuer
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cert-manager-controller-approve:my-issuer-example-com # edit
+  name: cert-manager-controller-approve:google-cas-issuer
 subjects:
-- kind: ServiceAccount
-  name: cert-manager
-  namespace: cert-manager
+  - kind: ServiceAccount
+    name: cert-manager
+    namespace: cert-manager
 ```
 
-#### RBAC Syntax
+{{% pageinfo color="info" %}}
+The syntax for `googlecasissuers.cas-issuer.jetstack.io/*` is detailed [here](#approval-api-rbac-syntax).
+{{% /pageinfo %}}
 
-When a user or controller attempts to approve or deny a CertificateRequest, the
-cert-manager webhook will evaluate whether it has sufficient permissions to do
-so. These permissions are based upon the request
-itself- specifically the request's IssuerRef:
+{{% pageinfo color="info" %}}
+The "resource type"
+[corresponds to](https://kubernetes.io/docs/reference/using-api/api-concepts/#standard-api-terminology)
+the lower-case and plural version of the "kind" field.
+{{% /pageinfo %}}
+
+[subject]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-subjects
+
+#### RBAC required for custom approvers
+
+When building your own approver controller, you will need to add RBAC roles so
+that your approver can set the `Approved=True` and `Denied=True` conditions to
+the CertificateRequests that are referencing either built-in types or external
+issuer types (such as the
+[`GoogleCASIssuer`](https://github.com/jetstack/google-cas-issuer) or the
+[`AWSPCAIssuer`](https://github.com/jniebuhr/aws-pca-issuer) type).
+
+Let us imagine that our approver needs to approve CertificateRequests that are
+targeting the built-in ACME issuer as well as the `AWSPCAIssuer`:
 
 ```yaml
-apiGroups: ["cert-manager.io"]
-resources: ["signers"]
-verbs: ["approve"]
-resourceNames:
- # namesapced signers
- - "<signer-resource-name>.<signer-group>/<signer-namespace>.<signer-name>"
- # cluster scoped signers
- - "<signer-resource-name>.<signer-group>/<signer-name>"
- # all signers of this resource name
- - "<signer-resource-name>.<signer-group>/*"
-```
```

Comment:
We have deleted this yaml snippet, however I find this much easier to read and understand as a kube user.

Is there a way we can include this fist, and then follow on with a worked example please?